site/documentation/content/admin-guide/secure_communication.md

```diff
@@ -49,7 +49,7 @@ The `demo-certs/certs` folder includes the following demo keys and certificates
 ### Device Registry
 
 The Device Registry supports the use of TLS for connections to protocol adapters and the Auth Server.
-Please refer to the [Device Registry admin guide]({{< relref "device-registry-config.md" >}}) for details regarding the required configuration steps.
+Please refer to the [Device Registry admin guide]({{< relref "file-based-device-registry-config.md" >}}) for details regarding the required configuration steps.
```

Comment:
please provide a separate section for both the *file based* and the *MongoDB based* registry

Reply:
@sophokles73 I have now included separate section for both the _file based_ and the _MongoDB based_ registry. Please take a look again.